test/integration/infra.go

```diff
@@ -66,9 +66,21 @@ type infra struct {
 	// sidecar initializer
 	UseInitializer bool
 	InjectConfig   *inject.Config
+
+	config model.IstioConfigStore
 }
 
 func (infra *infra) setup() error {
+	crdclient, crderr := crd.NewClient(kubeconfig, model.IstioConfigTypes, "")
```

Comment:
nit: why `crderr`? we don't use it in any special way

Reply:
Linter explains about shadowed `err` down below.

Reply:
Please fix linter instead.